FrontEnd/diary-ui/src/components/Login.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export default function Login() {
+    return(
+        <form>
+        <h2>Login</h2>
+        <fieldset>
+          <ul>
+            <div>
+              <label for="username">Username </label>
+              <input type="text" id="username" placeholder="Enter username or email" required/>
+            </div> 
```

Comment:
Since we are using same input format for each field we can use a common input component and pass `type`, `for` and `placeholder` as props. We will not need id because we will be using react to control our component.